### PR TITLE
Add my mods

### PR DESCRIPTION
**Pull request by hantabaru1014**
Added the following three my mods.
- ReleaseSRanipalOnQuit
- SaveItemTweaks
- NeosBetterIMESupport

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -5054,6 +5054,78 @@
...
+                        {
+                            "url": "https://github.com/hantabaru1014/ReleaseSRanipalOnQuit/releases/download/v1.0.0/ReleaseSRanipalOnQuit.dll",
+                            "filename": "ReleaseSRanipalOnQuit.dll",
+                            "sha256": "e395268564f25ed48ec28a113d5e89a1035ea3171e54413942ddc4562deb1ea3"
+                        }
+                    ]
+                }
+            }
+        },
+        "net.hantabaru1014.SaveItemTweaks": {
```
Comment (XDelta) [inline manifest.json:5081]: Built in debug mode

File: manifest.json [hunk 2]
```diff
@@ -5054,6 +5054,78 @@
...
+                        {
+                            "url": "https://github.com/hantabaru1014/SaveItemTweaks/releases/download/v1.0.0/SaveItemTweaks.dll",
+                            "filename": "SaveItemTweaks.dll",
+                            "sha256": "f4f11529a3556613e1e6e7b16c1fa06bf33acda1ecb56882f871e75813de800f"
+                        }
+                    ]
+                }
+            }
+        },
+        "net.hantabaru1014.NeosBetterIMESupport": {
```
Comment (XDelta) [inline manifest.json:5105]: Built in debug mode

File: manifest.json [hunk 3]
```diff
@@ -5054,6 +5054,78 @@
                 }
             }
         },
+        "net.hantabaru1014.ReleaseSRanipalOnQuit": {
```
Comment (XDelta) [inline manifest.json:5057]: Looks good

File: manifest.json (hunk 1 above)
Comment (hantabaru1014) [reply manifest.json:5081]: Thanks for your review!
I added v1.0.1 built in release mode.

File: manifest.json (hunk 2 above)
Comment (hantabaru1014) [reply manifest.json:5115]: Also added v1.0.1

**Review by XDelta — APPROVED** [review]
Looks all good to me

**Pull request merged**